.github/workflows/iam-validate.yml

```diff
@@ -0,0 +1,22 @@
+# Workflow that validates AOD IAM request.
+name: 'iam-validate'
+
+on:
+  pull_request:
+    branches:
+      - 'main'
+    paths:
+      - 'iam.yaml'
+
+concurrency:
+  group: '${{ github.workflow }}-${{ github.head_ref || github.ref }}-iam-request-validation'
+  cancel-in-progress: true
+
+jobs:
+  validate:
+    uses: 'abcxyz/access-on-demand/.github/workflows/iam-validate.yml@main' # Currently using main, change here to ping the tag/ref.
```

Comment:
Notice the CLI validation will fail if iam.yaml does't exist.

Reply:
fixing it in the aod repo, thanks!